lib/link.js

```diff
@@ -39,6 +39,11 @@ export default class Link extends Component {
       return
     }
 
+    if (typeof href === 'object') {
```

Comment:
Since `typeof null === 'object'` results in `true` and `require('url').format(null)` throws an exception, should we do `if(href && typeof href === 'object')`?